tests/apps/core/test_core_models_users.py

```diff
@@ -72,6 +96,22 @@ def test_models_users_username_ascii(self):
             ],
         )
 
+    def test_models_users_email_empty(self):
+        """The "email" field can be empty."""
+        UserFactory(email="")
+
+    def test_models_users_email_unique(self):
+        """The "email" field is not unique."""
+        user = UserFactory()
+        UserFactory(email=user.email)
+
+    def test_models_users_email_normalization(self):
+        """The email field should be automatically normalized upon saving."""
+        user = UserFactory()
+        user.email = "Thomas.Jefferson@Example.com"
+        user.save()
+        self.assertEqual(user.email, "Thomas.Jefferson@example.com")
```

Comment:
No need for `user.refresh_from_db()` here ?

Reply:
no no because it's done in the save.